packs/src/tables/wand-of-wonder.json

```diff
@@ -43,7 +43,7 @@
       "drawn": false,
       "documentCollection": "",
       "_id": "GGNQJzAVEiDumXq2",
-      "text": "You are stunned until the start of your next turn, believing something awesome just happened.",
+      "text": "You are @Compendium[dnd5e.rules.w7eitkpD7QQTB6j0.JournalEntryPage.ZyZMUwA2rboh4ObS]{Stunned} until the start of your next turn, believing something awesome just happened.",
```

Comment:
I don't believe these old-style content links will work when linking to an embedded Document inside a compendium (they will only open the parent JournalEntry but will not go to the correct page). I think you have to use the new `@UUID` content links.

```
@UUID[Compendium.dnd5e.rules.w7eitkpD7QQTB6j0.JournalEntryPage.ZyZMUwA2rboh4ObS]{Stunned}
```

Reply:
That explains why I could not reach the correct page.  Thanks.  I will get on it